05-Jason-Luo/wk02/4-thu/aboutme-jquery/js/aboutme.js

```diff
@@ -0,0 +1,81 @@
+// // var fontStyle = document.getElementsByTagName("body").style.fontFamily = "Arial,sans-serif";
+
+
+
+// var spanNick = document.getElementsByClassName("nickname");
+// var spanFavor = document.getElementsByClassName("favorites");
+// var spanHome = document.getElementsByClassName("hometown");
+
+// var i;
+// for (i = 0; i < spanNick.length; i++) {
+//     spanNick[i].textContent = spanNick[i].textContent.replace(" ");
+// }
+
+// var y;
+// for (y = 0; y < spanFavor.length; y++) {
+//     spanFavor[y].textContent = spanFavor[y].textContent.replace(" ");
+// }
+
+// var z;
+// for (z = 0; z < spanHome.length; z++) {
+//     spanHome[z].textContent = spanHome[z].textContent.replace(" ");
+// }
+
+// // Iterate through each li and change the class to "listitem". Add a style tag that sets a rule for "listitem" to make the color red.
+
+// var listClass = document.getElementById("li").className = "listitem";
+// var colorChange = listClass.fontStyle('red');
+
+// Create a new img element and set its src attribute to a picture of you. Append that element to the page.
+
+
+//Alt solution
+// Create a new Javascript file and link to it with a script tag at the bottom.
+
+// (In JS) Change the body tag's style so it has a font-family of "Arial, sans-serif".
+
+const $body = $("body");
+// document.body.style.fontFamily = 'Arial, sans-serif';
+$body.css({ "fontFamily": "Arial,sans-serif" });
+
+
+// (In JS) Replace each of the spans (nickname, favorites, hometown) with your own information.
+
+// const nickname = document.getElementById('nickname');
+const $nickname = $('#nickname').eq(0);
+// console.log($nickname);
+
+// nickname.innerHTML = 'The Blade';
+$nickname.html('The Blade');
```

Comment:
I would use the `.text()` method except in cases where I'm specifically trying to insert html. This prevents you from unintentionally inserting HTML into your page which can drastically (and confusingly) disrupt its look and layout.
```suggestion
$nickname.text('The Blade');
```